pkg/virt-api/rest/subresource.go

```diff
@@ -608,9 +619,15 @@ func (app *SubresourceAPIApp) StopVMRequestHandler(request *restful.Request, res
 		return
 	}
 
+	var oldGracePeriodSeconds int64
```

Comment:
nit: this can be declared inside the `if` statement

Reply:
+1

Reply:
oldGracePeriodSeconds is used in the RunStrategyHalted switch case outside of this if statement